### DISCUSSION TITLE
8BitDo Pro 2 support

### DISCUSSION THREAD

**Discussion by Sharkeology** [darthcloud/BlueRetro#683] [General]
Has anyone else attempted to use the Pro 2 controller with the BlueRetro project? I'm finding that I'm able to connect to the ESP32 over Bluetooth via any of the modes (Switch, Android, Dinput and Xinput) but no button presses are registered. I'm wondering whether the controller does things differently compared to others and whether this has been dealt with in the past.

**Comment by Invictaz** [comment]
Please add HCI Snoop log via either Ubuntu or Windows 10. There is an entry on the wiki how to do it.
